### PR TITLE
fix/constructor change constructor for tests

### PR DESCRIPTION
**Pull request by ZelikSV**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,9 +1,14 @@
 class MyArray {
-  constructor(...a) {
-    for (let i = 0; i < a.length; i++) {
-      this[i] = a[i];
+  constructor(...args) {
+    if (args.length === 1 && typeof args[0] === 'number') {
```
Comment (Vladyslav-Shvorobuk) [inline src/index.js:3]: Why typeof args[0] must be only number ???

**Review by ghost — APPROVED** [review]
that looks ok

**Review by Vladyslav-Shvorobuk — APPROVED** [review]

File: src/index.js (hunk 1 above)
Comment (ZelikSV) [reply src/index.js:3]: https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array

**Review by Arkant — APPROVED** [review]
approved

**Pull request merged**